backends/platform/libretro/dist/README.md

```diff
@@ -0,0 +1,5 @@
+Roland_SC-55.sf2 - Copyright (c) 2015 deemster
```

Comment:
What is the license for this file?

Reply:
That's indicated in the same file, few lines down. I've changed to be in a single line for clarity.